linera-chain/src/manager.rs

```diff
@@ -600,6 +610,7 @@ impl From<&ChainManager> for ChainManagerInfo {
             current_round,
             leader: manager.round_leader(current_round).cloned(),
             round_timeout: manager.round_timeout,
+            pending_blobs: manager.pending_blobs.clone(),
```

Comment:
Maybe we should leave this empty here, and only add them together with the other values if explicitly requested.